Vindicta.Altis/UI/BuildUI/BuildUI_Objects.h

```diff
@@ -98,6 +98,9 @@ class BuildObjects
 			// Lights
 			__BUILD_OBJECT_CLASS(Light0,Land_TentLamp_01_standing_F, 10)
 			__BUILD_OBJECT_CLASS(Light1,Land_LampShabby_F, 20)
+			__BUILD_OBJECT_CLASS(Light2,Campfire_burning_F, 5)
```

Comment:
We can't have cost which is not an integer multiple of build resource amount per build resource item

Reply:
Yeah there are many places like that right now in this PR